internal/config/controller.go

```diff
@@ -252,6 +257,10 @@ func GetEnvComponentValues(config *s2hv1beta1.ConfigSpec, compName string, envTy
 				"cannot get values file of %s env from url %s", envType, url)
 		}
 
+		valuesBytes = []byte(template.TextRender("TeamNameRendering",
+			string(valuesBytes),
+			teamObject{ TeamName: teamName },
+		))
```

Comment:
I think we need to do the template render for the env config URLs, not in the values file